R/tf_estimator.R

```diff
@@ -275,8 +275,6 @@ evaluate.tf_estimator <- function(object,
 #'
 #' Save an estimator (alongside its weights) to the directory `export_dir_base`.
 #'
-#' TODO: we should document how a user can load a saved model here as well.
```

Comment:
Is this going to be documented in tfserver?

Reply:
Right, we do have a `load_savedmodel()` function in `tfdeploy`, not sure if that will be the ultimate place where this function is made available, but we would document that wherever this ends.